### PR TITLE
MWPW-147239 use federal for checkout-links too

### PR DESCRIPTION
**Pull request by npeltier**
will send numbers & more details when related PR https://github.com/adobecom/milo/pull/2187 is merged 

Resolves: [MWPW-147239](https://jira.corp.adobe.com/browse/MWPW-147239)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.live/?martech=off
- After: https://MWPW-147239--milo--npeltier.hlx.live/?martech=off

### CODE REVIEW

**Review by mokimo — APPROVED** [review]
Coolio

**Comment by npeltier** [comment]
well not so cool @mokimo :) 
i think it's better using configuration, but i can't do that until we fix https://github.com/orgs/adobecom/discussions/2220 
ideal would be here that default configuration points to federal and i let code exactly like this.

what is clear is that PR will not bring any perf gain (actually perf will be slightly worse on lower env due to DNS fetch).

File: libs/blocks/merch/merch.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {
 import { replaceKey } from '../../features/placeholders.js';
 
 export const PRICE_LITERALS_URL = 'https://milo.adobe.com/libs/commerce/price-literals.json';
-export const CHECKOUT_LINK_CONFIG_PATH = '/commerce/checkout-link.json'; // relative to libs.
+export const CHECKOUT_LINK_URL = 'https://www.adobe.com/federal/commerce/checkout-link.json'; // relative to libs.
```
Comment (3ch023) [inline libs/blocks/merch/merch.js:7]: as GWP, how do I validate my change before publishing it live?
should we use https://www.stage.adobe.com/federal/commerce/checkout-link.json?
Does it point to hlx.page content?
cc @mokimo

**Review by 3ch023 — CHANGES_REQUESTED** [review]
the current change takes away any ability to test changes to the file.
imo before merging we should address this and make checkout-links changes testable by GWP content QA before they publish their changes

File: libs/blocks/merch/merch.js (hunk 1 above)
Comment (mokimo) [reply libs/blocks/merch/merch.js:7]: no, stage.adobe.com is not yet connected to stage/federal/hlx.page. I'd recommend to use stage--federal--adobecom.hlx.page/commerce/checkout-link.json
https://github.com/orgs/adobecom/discussions/2220#discussioncomment-9260878 

until we properly set up stage as part of the helix-5 migration

**Comment by npeltier** [comment]
closing for now

**Pull request closed without merging**